### PR TITLE
Create explicit LICENSE file

### PR DESCRIPTION
**Pull request by d3zd3z**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice,
```
Comment (jenswikl) [inline LICENSE:7]: Since the license is referring to copyright notice above I guess we should add one also to avoid ambiguity.

**Comment by d3zd3z** [comment]
I copied in the full text. I also made this owned by my Linaro email address, instead of personal.

File: LICENSE (hunk 1 above)
Comment (d3zd3z) [inline LICENSE:7]: Added, pulled in the whole text.

**Comment by jenswikl** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by pascal-brand38** [comment]
`Reviewed-by: Pascal Brand <pascal.brand@linaro.org>`

**Comment by jockebech** [comment]
@d3zd3z , could you please update the commit with the tags and add a Signed-off-by tag?

**Comment by d3zd3z** [comment]
Pushed with headers and sob.

**Pull request merged**